libs/vectorise/include/vectorise/arch/avx2/math/abs.hpp

```diff
@@ -26,25 +26,29 @@ namespace vectorise {
 inline VectorRegister<float, 256> abs(VectorRegister<float, 256> const &a)
 {
   const __m256 sign = _mm256_castsi256_ps(_mm256_set1_epi32(1 << 31));
-  return VectorRegister<float, 256>(_mm256_andnot_ps(sign, a.data()));
+  auto const   ret  = VectorRegister<float, 256>(_mm256_andnot_ps(sign, a.data()));
```

Comment:
What prompted these changes?

Reply:
The modernize-return-braced-init-list check enforces brace initialisation rather than repeating the return type in a return statement. However, you have to spell out the type here, as the ctor is explicit. This change keeps both checks happy.